.ruff.toml

```
@@ -0,0 +1,373 @@
+target-version = "py38"
+line-length = 88
+select = ["ALL"]
+exclude=[
+    "astropy/extern/*",
+    "*_parsetab.py",
+    "*_lextab.py"
+]
+extend-ignore = [
```

[`extend-ignore` should be replaced with `ignore`](https://beta.ruff.rs/docs/settings/#extend-ignore): 
```suggestion
ignore = [
```